src/MainWindow.vala

```diff
@@ -839,15 +840,44 @@ namespace Terminal {
             }
         }
 
-        private void update_context_menu () {
-            clipboard.request_targets (update_context_menu_cb);
```

Comment:
It doesn't seem intentional to remove this. If it is, then `update_context_menu_cb` should also be removed since it is now never used